lib/net.cpp

```diff
@@ -15,6 +15,25 @@
 
 // https://wiki.openssl.org/index.php/SSL/TLS_Client
 
+static std::string error_to_string(int err)
+{
+#ifdef _WIN32
+    static char msgbuf[256] = { '\0' };
```

Comment:
Why is this declared static? It doesn't need to be static.

Reply:
Removed static.